src/app/pages/BuyPage/components/SwapAssetSelector/styled.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components/macro';
```

Comment:
we don't need this if we use one of two global components.

Reply:
fixed